src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/VisitorRepository.java

```diff
@@ -8,5 +8,5 @@
 
 @Repository
 public interface VisitorRepository extends PagingAndSortingRepository<VisitorInformation, String> {
-    List<VisitorInformation> findAllByVisitIdAndBookingId(Long visitId, Long bookingId);;
+    List<VisitorInformation> findAllByVisitId(Long visitId);
```

Comment:
Visit Id is a primary key on the visits table, so there's no need to also use the booking id to identify a visitor.